loleaflet/src/control/Permission.js

```diff
@@ -146,7 +146,7 @@ L.Map.include({
 	},
 
 	isPermissionEditForComments: function() {
-		return true;
```

Comment:
Ideally it would be great to have a comment here to explain what is going on (that we allow commenting even when in the view mode on the mobile) - can you do please?